tests/integration/components/hyper-table-v2/filtering-renderers/numeric-test.ts

```diff
@@ -140,25 +140,28 @@ module('Integration | Component | hyper-table-v2/filtering-renderers/numeric', f
 
     test('it triggers applyFilter when the range values are changed', async function (assert: Assert) {
       const handlerSpy = sinon.spy(this.handler);
+      //@ts-ignore
+      Ember.run.debounce = function (target: () => {}, func: () => {}) {
+        func.call(target);
+      };
```

Comment:
Not sure to understand this part. Why define that?

Reply:
Was wondering why this specific test took ~1300ms to run. Turns out the input of the component has a debouncer set to 500. (And it is triggered twice in the test)
So instead of waiting 500ms for nothing, i'm stubbing the debouncer to run it directly.

Reply:
smart !